src/System.Net.Requests/tests/FileWebRequestTest.cs

```diff
@@ -109,10 +108,8 @@ public abstract class FileWebRequestTestBase
                 using (WebResponse response = await GetResponseAsync(request))
                 {
                     Assert.Equal(data.Length, response.ContentLength);
-                    Assert.Equal(data.Length.ToString(), response.Headers[HttpRequestHeader.ContentLength]);
 
                     Assert.Equal("application/octet-stream", response.ContentType);
-                    Assert.Equal("application/octet-stream", response.Headers[HttpRequestHeader.ContentType]);
```

Comment:
Why do these asserts need to be removed?  We're populating the content length and type in core but not in desktop?